.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,65 +1,42 @@
 <!--
-### Requirements
-
-Filling out the template is required.  Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.  All new code requires documentation and tests to ensure against regressions.
+Filling out this template is required.  Any PR that does not include enough information to be reviewed may be closed at a maintainers' discretion.  All new code requires documentation and tests to ensure against regressions.
 -->
 
 ### Description of the Change
-
-<!--
-We must be able to understand the design of your change from this description.  If we can't get a good idea of what the code will be doing from the description here, the pull request may be closed at the maintainers' discretion.  Also including any benefits that will be realized by the code change will be helpful.  Keep in mind that the maintainer reviewing this PR may not be familiar with or have worked with the code here recently, so please walk us through the concepts.  Please include screenshots (if appropriate).
--->
-
-<!-- Enter any applicable Issues here. Example: -->
-Closes #
-
-### Alternate Designs
-
-<!-- Explain what other alternates were considered and why the proposed version was selected. -->
-
-### Possible Drawbacks
-
-<!-- What are the possible side-effects or negative impacts of the code change? -->
-
-### Verification Process
-
 <!--
-What process did you follow to verify that your change has the desired effects?
-
-- How did you verify that all new functionality works as expected?
-- How did you verify that all changed functionality works as expected?
-- How did you verify that the change has not introduced any regressions?
+We must be able to understand the design of your change from this description.  The maintainer reviewing this PR may not have worked with this code recently, so please provide as much deatil as possible.
 
-Describe the actions you performed (e.g., commands you ran, text you typed, buttons you clicked) and describe the results you observed.
+Where possible, please also include:
+- verification steps to ensure your change has the desired effects and has not introduced any regressions
```

Comment:
I like getting rid of the `Alternate Designs` and `Possible Drawbacks` sections, as those always confused me a little.

I may have been using it wrong but the `Verification Process` section I typically would use to put test instructions. May be worth adding a `How to test changes` section

Reply:
I agree that we should have a dedicated section for testing instructions. Having a testing section will encourage contributors to provide detailed testing steps.

> I like getting rid of the `Alternate Designs` and `Possible Drawbacks` sections, as those always confused me a little.

I don't use these sections too 😅.